packages/Sandblocks-Core/SBBlock.class.st

```diff
@@ -1239,6 +1286,12 @@ SBBlock >> isDeclaration [
 	^ false
 ]
 
+{ #category : #'as yet unclassified' }
+SBBlock >> isDragging [ 
+	
+	self withDecorator: SBMoveDecorator do: [:decorator | ^ decorator isDragging] ifAbsent: [^ false]
```

Comment:
same nitpick about single return as above :) (withDecorator:do:ifAbsent: returns the value that is computed in the block that it executes)